articles/configuration/msbuild.md

```diff
@@ -1,3 +1,41 @@
-# MSBuild configuration
+---
+title: MSBuild Configuration
+author: uniqueiniquity
+ms.author: belichtm
+ms.date: 11/13/2017
+ms.topic: article
+ms.prod: .net
+uid: msbuild
+---
 
-TODO
+# MSBuild Configuration
+
+## Props and Targets
+
+There are two different sources of information that tell MSBuild how to handle TypeScript. The first, `Microsoft.TypeScript.Default.props`, establishes some default TypeScript compilation settings. The second, `Microsoft.TypeScript.targets`, gives instructions on how to handle TypeScript files as part of the building and cleaning processes. To include these in your project, you will need to (in most cases) import `Microsoft.TypeScript.Default.props` at the beginning of your project file and `Microsoft.TypeScript.targets` at the end, as shown [in the TypeScript handbook](http://www.typescriptlang.org/docs/handbook/integrating-with-build-tools.html#msbuild).
+
+> [!NOTE]
+> ASP.NET Core projects implicitly include these files, so an ASP.NET Core project file will not need to import them.
+
+## TypeScript Tools Version
+Since a particular machine may have multiple versions of TypeScript installed, MSBuild allows you to set the TypeScriptToolsVersion property in your project file to specify which version to use.
+If this property is not set, MSBuild defaults to using the newest version of TypeScript that is installed. However, if you are using the TypeScript NuGet package instead, its version will override any setting in the project file.
+
+## Input Files
+There are two ways that files can be recognized by and therefore built with the project: either by the project file itself or by way of a tsconfig.json file.
```

Comment:
Sounds a bit ambiguous, how about "There are two ways of including TypeScript files in the compilation: either using a tsconfig.json or using the `TypeScriptCompile` MSBuild itemtype." (Then instead of saying "without TSConfig" in the header below, you would say "TypeScriptCompile" or something)

Reply:
Changed.